compile.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/bash
```

Comment:
On the topic of mac:

```suggestion
#!/usr/bin/env bash
```